data/blog/battery-life.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: 'Battery life instrumentation'
+date: '2023-04-25'
+tags:
+  - 'oss'
+  - 'mobile'
+authors: ['mihai']
+images: ['/images/battery-life/social.png']
+summary: 'Instrumentation that we’re adding to the Tailscale clients to have a scalable approach to battery life investigations.'
```

Comment:
For consistency's sake, I'd prefer this to be more of a full sentence. Maybe:

We've added instrumentation to the Tailscale clients to make our investigations into batter life issues more scalable.

Or something like that?